src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+$darkBlue: #060b35;
+$gray: #616070;
+$mainColor: #00acdc;
+$white: #fff;
```

Comment:
You can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`).

Fix the variable names, they should show their purpose, not their content.  if I decided to change the color, I would have to change it everywhere, not just in the variable definition.